tests/unit/test_operations/test_dataset_column_order.py

```diff
@@ -1,16 +1,20 @@
 from unittest.mock import MagicMock
 
 import pandas as pd
+from cdisc_rules_engine.models.dataset.dask_dataset import DaskDataset
+from cdisc_rules_engine.models.dataset.pandas_dataset import PandasDataset
 
 from cdisc_rules_engine.models.operation_params import OperationParams
 from cdisc_rules_engine.operations.dataset_column_order import DatasetColumnOrder
+import pytest
 
 
-def test_get_column_order_from_dataset(operation_params: OperationParams):
+@pytest.mark.parametrize("dataset_type", [(PandasDataset), (DaskDataset)])
```

Comment:
Will do. Just a heads up my next update will include that change as well as some changes I made to the dataset implementations. The big change is just that now anytime an implementation returns a single column, it is returned as a pandas series. I am ok with this change because I find it hard to imagine a single column exceeding memory limits. This also makes it easier to reuse the same logic for all check operators